docs/developer.md

```diff
@@ -80,6 +80,24 @@ For accessing Vault the `VAULT_ADDR` env var has to be set, Pipeline stores JWT
 export VAULT_ADDR=http://127.0.0.1:8200
 ```
 
+#### Start Pipeline
+
+Once you have the docker containers running for the development environment, you should be able to start the pipeline platform.
+
+You can install and then run it with:
+```bash
+$ go install ./cmd/pipeline/
```

Comment:
This should rather be `make build` and `build/pipeline`. (Event better: mention that both VSCode and Goland has builtin tasks for this)

Reply:
Made the changes, thx!